content/docs/gke/private-clusters.md

```diff
@@ -80,7 +84,7 @@ export PROJECT_NUMBER=$(gcloud projects describe ${PROJECT} --format='value(proj
                            dns.googleapis.com  --project=${PROJECT}
     ```
 
-1. Check if you have an access policy object already created:
+2. Check if you have an access policy object already created:
```

Comment:
Why are you explicitly changing the numbering rather than letting markdown auto enumerate it for you?

Reply:
@oxxygenio just saw your comment about numbering. I would suggest reverting the numbering changes and then we should try to figure out why the numbering is broken

Reply:
FYI, here's the preview of your PR, where you can see how the page looks with your changes included, when built by Netlify for the doc website:
https://deploy-preview-1563--competent-brattain-de2d6d.netlify.com/docs/gke/private-clusters/
Netlify builds the preview each time you push a commit up to GitHub.